cognite/powerops/client/shop/shop_run_api.py

```diff
@@ -73,7 +74,8 @@ def trigger_case(self, case: Case, shop_version: str) -> tuple[list, list[SHOPRu
                     source="DayaheadTrigger",
                 )
             )
-        self._cdf.events.create([new_event.as_cdf_event() for new_event in shop_events])
+
+        [create_event(self._cdf, new_event.as_cdf_event()) for new_event in shop_events]

```

Comment:
if this list isnt used anywhere, why create it?

Reply:
I'll just revert this back to how it used to be. In this method we aren't allowing the event_id to be set - it would become a bit messy if we did.